Registry/prefab.tools.setreg

```diff
@@ -7,17 +7,16 @@
             {
                 "Processing":
                 {
-                    "Stack":
-                    {
-                        "PlayInEditor":
+                    "Stack": {
+                        "PlayInEditor": 
```

Comment:
Nitpick: Opening brace should go to its own line. (Since there are only whitespace changes here, consider reverting this file to it's original version.)